theseus/geometry/manifold.py

```diff
@@ -81,6 +81,14 @@ def project(
     ) -> torch.Tensor:
         return self._project_impl(euclidean_grad, is_sparse)
 
+    @staticmethod
+    def normalize(data: torch.Tensor) -> torch.Tensor:
```

Comment:
Is normalize supposed to be `@abc.abstractmethod` with the `Manifold` version doing `pass`?

Reply:
Same question for `_data_check`.